src/__stories__/badge-story.tsx

```diff
@@ -22,9 +22,13 @@ export const Default: StoryComponent<Args> = ({inverse, value}) => {
         <ResponsiveLayout fullWidth isInverse={inverse}>
             <Box padding={16} width="fit-content" dataAttributes={{testid: 'content'}}>
                 <Badge value={value !== 'undefined' ? +value : undefined}>
-                    <IconButton onPress={() => {}} aria-label="Read notifications">
+                    <Touchable
+                        style={{display: 'inline-block', width: 24, height: 24}}
+                        onPress={() => {}}
+                        aria-label="Read notifications"
+                    >
                         <IconBellFilled />
-                    </IconButton>
+                    </Touchable>
```

Comment:
Yes, we could have done that, but the badge would look "far" from the icon (because of the IconButton circle container). I believe it's better to keep it as it is, because it shows how the badge renders "on top" on the icon

Without hover:

![image](https://github.com/Telefonica/mistica-web/assets/25785151/2926b23c-2d14-4970-a154-4e706a82b1eb)

With hover:

![image](https://github.com/Telefonica/mistica-web/assets/25785151/25a1b5a3-db9b-4d00-b2f5-1e11e8b75565)